### PR TITLE
Add Typography

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

**Review by danieledamiani — APPROVED** [review]

File: packages/tempus-client_v3/src/components/Typography/Typography.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  | 'subtitle'
+  | 'body-primary'
+  | 'body-secondary'
+  | 'boyd-tertiary';
+
+type TypographyColor = 'text-primary';
+
+type TypographyWeight = 'regular' | 'medium' | 'bold';
+
+type TypographySpacing = 'regular' | 'mono';
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/Typography/Typography.tsx:17]: I think that a better name would be `TypographyType`, `TypographyFamily` or something similar. `Spacing` reminds me of letter spacing.

File: packages/tempus-client_v3/src/components/Typography/Typography.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+import parse from 'html-react-parser';
+
+type TypographyVariant =
+  | 'header'
+  | 'subheader'
+  | 'title'
+  | 'subtitle'
+  | 'body-primary'
+  | 'body-secondary'
+  | 'boyd-tertiary';
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/Typography/Typography.tsx:11]: Typo

File: packages/tempus-client_v3/src/components/Typography/Typography.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/Typography/Typography.tsx]: Fixed.

File: packages/tempus-client_v3/src/components/Typography/Typography.tsx (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v3/src/components/Typography/Typography.tsx]: Fixed.

**Review by iamvukasin — APPROVED** [review]

**Pull request merged**